project_cost_report/report/report.py

```diff
@@ -247,30 +251,30 @@ def _get_time_categories(self, project, report_context):
             for c in sorted_categories
         ]
 
-    def _get_time_total(self, project):
-        lines = self._get_timesheet_analytic_lines(project)
+    def _get_time_total(self, projects):
+        lines = self._get_timesheet_analytic_lines(projects)
         total_amount = sum(l.amount for l in lines)
         return float_round(-total_amount, 2)
 
-    def _get_time_total_hours(self, project):
-        lines = self._get_timesheet_analytic_lines(project)
+    def _get_time_total_hours(self, projects):
+        lines = self._get_timesheet_analytic_lines(projects)
         total_hours = sum((l.unit_amount or 0) for l in lines)
         return float_round(total_hours, 2)
 
-    def _get_timesheet_analytic_lines(self, project):
-        return project.analytic_account_id.line_ids.filtered(
+    def _get_timesheet_analytic_lines(self, projects):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/Numigi/odoo-project-addons/pullRequest?prid=6144302)